src/components/common/ModalComponent.jsx

```diff
@@ -12,20 +13,24 @@ export default class ArtModal extends Component {
     const { children } = this.props;
     const childrenWithProps = React.Children.map(children, child =>
       React.cloneElement(child,
-        { toggleModal: this.toggleModal }
+        { }
```

Comment:
Please confirm with @HawiCaesar where the `toggleModal` prop is being used to ensure this change doesn't break existing functionality

Reply:
The `toggleModal` prop is used when you add a `Cancel` button onto the modal to allow the user to close the modal.

Reply:
@HawiCaesar Ok, but I noticed a React Error with `line 15`

Reply:
@andela-fmustapha , I see your point. I think Emeka had asked whether we should have an `x` icon or `Cancel` button as having both may be redundant. Eric chimed in as well and said indeed it is redundant. He also said `if we have the cancel button we can do away with the close icon`

Looking closer at your code, I see you added a loop that would return a list of elements. This is okay as well. To explain further, when the modal was implemented, the idea was to place a component at it. like so
```
<ArtModal>
  <YourComponent {...props} />
</ArtModal>
```
Why this approach ? So that you can customize your component as you see fit.

Reply:
You can try pass a component like so 
```
<ArtModal>
  <YourComponent {...props} />
</ArtModal>
```
and not implement the `Cancel` button. This feature has no particular UI design to follow so having the x icon only as you had planned should not be a problem as it comes with the modal itself.

Reply:
@HawiCaesar @andela-fmustapha What's your decision? Are you having a list of elements or passing element as prop?
In my personal opinion, I prefer passing any element as prop as explained by Hawi